docs/src/index.md

```diff
@@ -1,32 +1,54 @@
-# Reference
-
-HSL MA57 and MA97 can be used for the solution of symmetric, possibly indefinite, linear systems.
-They are often used for the solution of saddle-point systems, i.e., systems of the form
-```math
-\begin{bmatrix}
-H & A^T \\
-A &
-\end{bmatrix}
-\begin{bmatrix}
-x \\ y
-\end{bmatrix}
-=
-\begin{bmatrix}
-b \\ c
-\end{bmatrix}.
-```
-Special cases occur when ``H`` is the identity matrix and either ``b = 0`` or ``c = 0``, which correspond to the least-squares problem
-```math
-\min_y \ \|A^T y - b\|
-```
-and to the least-norm problem
-```math
-\min_x \ \|x\| \ \text{subject to } Ax = c
+# [HSL.jl documentation](@id Home)
+
+A set of interfaces to [HSL](http://www.hsl.rl.ac.uk) packages for sparse linear algebra.
+
+Certain HSL packages are freely available to all, others are freely available
+to academics only. Please refer to the website above for licensing information.
+In all cases, users are responsible for obtaining HSL packages.
+
+## Installing
+
+```JULIA
+julia> ]
+pkg> add HSL
 ```
-respectively.
 
-```@autodocs
-Modules = [HSL]
-Private = false
-Order   = [:module, :constant, :type, :function, :macro]
```

Comment:
The index is empty now.

Reply:
This is probably because the docstrings are not loaded if HSL is not build properly. I will move them to the main file HSL.jl

Reply:
I have a similar issue for NLPModelsKnitro https://github.com/JuliaSmoothOptimizers/NLPModelsKnitro.jl/pull/97

Reply:
Since this problem already existed, I suggest to open an issue and fix it separately.